build_modules/lib/src/kernel_builder.dart

```diff
@@ -236,18 +237,27 @@ void _addRequestArguments(
     multiRootScheme,
     '--exclude-non-sources',
     summaryOnly ? '--summary-only' : '--no-summary-only',
+    '--libraries-file',
+    p.join(sdkDir, 'lib', 'libraries.json'),
+    '--reuse-compiler-result',
+    '--use-incremental-compiler',
   ]);
+  request.inputs.add(Input()
+    ..path = '${Uri.file(p.join(sdkDir, sdkKernelPath))}'
+    ..digest = [0]);
```

Comment:
Maybe add a comment that this file will never change without invalidating the entire worker